src/layers/VivViewerLayer/VivViewerLayerBase.js

```diff
@@ -14,7 +14,10 @@ const defaultProps = {
   minZoom: { type: 'number', value: 0, compare: true },
   maxZoom: { type: 'number', value: 0, compare: true },
   renderSubLayers: { type: 'function', value: renderSubLayers, compare: false },
-  opacity: { type: 'number', value: 1, compare: true }
+  opacity: { type: 'number', value: 1, compare: true },
+  colormap: { type: 'string', value: '', compare: true },
+  dtype: { type: 'string', value: '<u2', compare: true },
```

Comment:
Have we defined the string keys for the different type dtypes? Are you pulling from numpy land here? Might be good to settle on convention.

Reply:
This is a `deck.gl` convention although it might be worth defining these constants in one location...